pages/docs/guides/validate.md

```diff
@@ -0,0 +1,123 @@
+
+Generate a sample `asyncapi.yaml` file (AsyncAPI document) with the following CLI command: 
+
+```
+asyncapi new --example=default-example.yaml --no-tty
+```
+
+# Validate AsyncAPI documents
+Validating an AsyncAPI document requires two different pieces:
+1. Validation against the specification.
+2. Validation against the best practices or company governance rules.
+
+## Validate against specification
+Let's discuss options for validating against the specification. (This ensures that every content of the document is written in accordance with the AsyncAPI specification.) 
+
+There are several tool options you may select for validating against the specification: _AsyncAPI Studio_, _AsyncAPI CLI_, and _Parsers_.
+
+### AsyncAPI Studio validation 
+[AsyncAPI Studio](https://github.com/asyncapi/studio#readme) provides a visual and easy way to validate your AsyncAPI documents against the specification. 
+
+(It uses the [AsyncAPI JavaScript parser](https://github.com/asyncapi/parser-js) behind the scenes to perform syntax checks and validate documents.)
+
+Errors in your document are highlighted with a red underline, showing which lines are invalid. The console also provides feedback, allowing you to further troubleshoot with detailed error messages.
+
+When a document is invalid, it provides the following error: `Empty or invalid document please fix errors / define AsyncAPI document`.
+
+### AsyncAPI CLI validation 
+The following [AsyncAPI CLI](https://github.com/asyncapi/cli#installation) command validates AsyncAPI documents in your local computer or in automation:
+
+ ```
+ asyncapi validate asyncapi.yaml
+
+ ```
+
+### Parsers (code) validation 
+AsyncAPI provides official [JavaScript](https://github.com/asyncapi/parser-js) and [Go](https://github.com/asyncapi/parser-go) parsers for validating the JSON schema in your AsyncAPI documents. 
+
+<Remember>
+If you do not run your schemas through the parsers, the JSON schemas alone cannot complete the entire validation. 
+
+Learn more about [custom JSON Schemas validation needs](https://github.com/asyncapi/spec-json-schemas#custom-validation-needs).
+</Remember>
+
+## Validation against company best practices or governance
+Now let's discuss options for validating against company best practices or governance.
+
+When AsyncAPI is used by various members of the team in an organization, API management policies can be applied to your messages before they arrive to your broker.
+
+One way this can be done is by using **Spectral**, an API linting tool which has a built-in [custom ruleset properties](https://meta.stoplight.io/docs/spectral/e5b9616d6d50c-custom-rulesets) with [AsyncAPI rules](https://meta.stoplight.io/docs/spectral/1e63ffd0220f3-async-api-rules) for the AsyncAPI specification; these help define best practices for your Event-Driven APIs. 
```

Comment:
```suggestion
One way this can be done is by using **Spectral**, an API linting tool which has a built-in [custom ruleset properties](https://meta.stoplight.io/docs/spectral/e5b9616d6d50c-custom-rulesets) with [AsyncAPI rules](https://meta.stoplight.io/docs/spectral/1e63ffd0220f3-async-api-rules) for the AsyncAPI specification. It also enables you to define company-specific rules that you can use internally. 
```